AnnoDesigner/Models/UserDefinedColor.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Diagnostics;
+using System.Runtime.Serialization;
+using AnnoDesigner.Core.Models;
+
+namespace AnnoDesigner.Models
+{
+    [DebuggerDisplay("{" + nameof(Type) + ",nq}")]
+    [DataContract]
+    public class UserDefinedColor : Notify
+    {
+        private UserDefinedColorType _type;
+        private SerializableColor _color;
+
+        [DataMember(Order = 0)]
```

Comment:
Do we need these `DataMember` and `DataContract` attributes? Especially as this is in settings anyway, so won't be normally seen by the user. I just feel like they add extra stuff that new developers may think are important when they are actually not.

Reply:
I explicitly added them to make it more obvious which properties would be serialized.
But I also can remove them.

Reply:
Yeah, I would just remove them.